app/assets/stylesheets/comments.scss

```diff
@@ -199,6 +199,21 @@ a.header-link {
       }
     }
   }
+  .editor-response-templates {
+    position: absolute;
+    left: 77px;
+    bottom: 4px;
+    button {
+      background: none;
+      border: none;
+      padding: 0;
+    }
+    &.in-reply {
+      z-index: 6;
+      left: 79px;
+      bottom: 2px;
```

Comment:
Had to do some weird z-index hackery here to let the button be clicked. Shouldn't be an issue though.